src/core/adapters/tara.js

```diff
@@ -0,0 +1,57 @@
+/**
+ * tara.ai adapter
+ *
+ * The adapter extracts the current task's identifier from the page url
+ * and uses the DOM to retrieve information for the corresponding ticket.
+ * Page urls follow the pattern:
+ *
+ *   https://app.tara.ai/<workspace-name>/tasks/<id>
+ *
+ * Each workspace has multiple projects (called "requirements"), task ids
+ * always start with `TASK-` followed by a 1-based index unique within the
+ * given workspace.
+ */
+
+import { match } from 'micro-match';
+
+import { $all, $has, $text } from './helpers';
+
+/**
+ * Tara has two ticket views, one in a modal-like overlay, one in full page view.
+ */
+function isModalView(doc) {
+  const modalViewSelector = 'input[data-cy="requirement-detail-name-input"]';
+  return $has(modalViewSelector, doc);
+}
+
+function extractDescription(doc) {
+  const description = [];
+  const editorRootNodes = $all('.DraftEditor-root', doc);
+
+  if (isModalView(doc)) editorRootNodes.shift(); // Skip project information editor node
+
+  const textNodes = $all('span[data-text="true"]', editorRootNodes[0]);
+  textNodes.forEach((node) => {
+    description.push(node.textContent);
+  });
+
+  return description.join('\n');
+}
+
+async function scan(loc, doc) {
+  if (loc.host !== 'app.tara.ai') return [];
+
+  const { index } = match('/:workspace/tasks/:index', loc.pathname);
+
+  if (!index) return [];
+
+  const id = $text('span[data-cy="task-modal-task-id-text"]', doc);
+  const description = extractDescription(doc);
+  const title = $text('textarea[data-cy="task-modal-task-name-text"]', doc);
+  const type = 'feature';
```

Comment:
Just outta curiosity: why did you decide to hardcode this to `feature`? I'd have expected it to be `task` if that's also their internal name.

Reply:
We've been using `feature` as a default so far if we can't extract a specific "type" (as per our README).

<img width="580" alt="readme" src="https://user-images.githubusercontent.com/706519/99649135-ba36cf00-2a4b-11eb-9ddd-5bdf9955c0cd.png">

I think this was inspired by the "feature-branching" conventions. Not sure whether this convention still makes sense or whether we should go with `task` here and remove the "(default)" from the README? @klappradla? @andreasknoepfle? @nickgnd?

Reply:
Plus, it felt like an odd repetition, given that `task` was already part of the `id` itself, i. e. `task/task-8`. I'd be happy to change this to something more expressive, but I couldn't come up with anything clever yet.

Reply:
> We've been using `feature` as a default so far if we can't extract a specific "type" (as per our README).

Perfectly fine 👍 

I tend to call everything I'm doing "chore", but that's probably just because I a) like the word while not knowing exactly what it means and b) because the work I'm doing feels more like 🚜